tapdb/assets_common.go

```diff
@@ -629,11 +633,28 @@ func parseGroupKeyInfo(tweakedKey, rawKey, witness, tapscriptRoot []byte,
 		}
 	}
 
+	// Parse group key version from database row.
+	groupKeyVersion, err := asset.NewGroupKeyVersionFromInt32(version)
+	if err != nil {
+		return nil, err
+	}
+
+	// Parse custom tapscript root if specified.
+	var customSubtreeRootHash fn.Option[chainhash.Hash]
+	if len(customSubtreeRoot) != 0 {
+		var rootHash chainhash.Hash
+		copy(rootHash[:], customSubtreeRoot)
+
+		customSubtreeRootHash = fn.Some(rootHash)
+	}
+
 	return &asset.GroupKey{
-		RawKey:        groupRawKey,
-		GroupPubKey:   *tweakedGroupKey,
-		TapscriptRoot: tapscriptRoot,
-		Witness:       groupWitness,
+		Version:             groupKeyVersion,
+		RawKey:              groupRawKey,
+		GroupPubKey:         *tweakedGroupKey,
+		TapscriptRoot:       tapscriptRoot,
+		Witness:             groupWitness,
+		CustomTapscriptRoot: customSubtreeRootHash,
```

Comment:
Should expand assertions for db tests to make sure these new fields are read/write properly. 

Although I don't think we need the new custom tapscript root at all. We can get away with just one field, the existing one, instead of having to lug around two fields where one is a transformation of the other.

Reply:
> Although I don't think we need the new custom tapscript root at all. We can get away with just one field, the existing one, instead of having to lug around two fields where one is a transformation of the other.

I think we need the custom tapscript root column so that we can read from it later to construct the group key reveal for the minting proof.